app/statemachine.js

```diff
@@ -476,8 +476,11 @@ CoreStateMachine.prototype.increasePlaybackTimer = function () {
 
       this.askedForPrefetch = false;
       this.pushState.bind(this);
```

Comment:
BTW this line does nothing. `bind()` does not call function, it just returns new function. 

So here is potential misbehavior, it does not behave as supposed to.

Reply:
```suggestion
      this.pushState();
```

most likely it should be just simple call.